specs/hivemq-openapi.yaml

```diff
@@ -40,7 +40,7 @@ info:
     \ can imported into popular API tooling (e.g. Postman) or can be used to generate\
     \ client-code for multiple programming languages.\n"
   title: HiveMQ REST API
-  version: 4.8.1
+  version: 4.16.0-SNAPSHOT
```

Comment:
I added a subtask to the 4.16.0 release ticket in order to assure this file is getting updated.